--- a/config/database.php
+++ b/config/database.php
@@ -1,6 +1,8 @@
 <?php

 use Illuminate\Support\Str;
+
+$pdoMysqlSslCa = class_exists(\Pdo\Mysql::class, false) ? \Pdo\Mysql::ATTR_SSL_CA : PDO::MYSQL_ATTR_SSL_CA;

 return [
 
@@ -61,7 +63,7 @@
             'engine' => null,
             'options' => extension_loaded('pdo_mysql') ? array_filter([
-                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
+                $pdoMysqlSslCa => env('MYSQL_ATTR_SSL_CA'),
             ]) : [],
         ],
 
@@ -81,7 +83,7 @@
             'engine' => null,
             'options' => extension_loaded('pdo_mysql') ? array_filter([
-                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
+                $pdoMysqlSslCa => env('MYSQL_ATTR_SSL_CA'),
             ]) : [],
         ],
